### DISCUSSION TITLE
Could you add XmlDoc to NuGet packages?

### DISCUSSION THREAD

**Discussion by nredko** [egramtel/tdsharp#153] [Q&A] [answered]
Thanks

**Comment by ForNeVeR** [comment, marked as answer]
Oh, I never noticed that we have no XmlDoc. Of course this is a good suggestion. I've open #154 about that, thanks for reporting!

**Reply by ForNeVeR** [reply]
Released v1.8.9.1 with the XML documentation.

**Comment by nredko** [comment]
Thank you very much!
It works.
